loadbalancer/src/main/java/com/io/loadbalancer/consistent_hashing/MonitorClient.java

```diff
@@ -0,0 +1,33 @@
+package com.io.loadbalancer.consistent_hashing;
+
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Objects;
+
+@Component
+public class MonitorClient {
+
+    public boolean getStatus(String host) {
+        try {
+            RestTemplate restTemplate = getTemplate();
+            String url = host + "/connectivityCheck";
```

Comment:
How does that endpoint differs from "/health"?

Reply:
And why do we check for anything more than the status code? The IM either works or not so having anything in body of that entpoint is useless to me.

Reply:
As you wrote above /health check only if IM works or not but it is possible that we have no origins on the IM but IM still works. In this case this endpoint return false.

Reply:
so the IM endpoint could just operate on status codes I guess?

Reply:
I think that there is no point in changing that right now, let's keep it as it is